[2025-10-03T13:12:07.755+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-03T13:12:07.778+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_four.select_rows_task scheduled__2025-10-03T13:10:00+00:00 [queued]>
[2025-10-03T13:12:07.786+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_four.select_rows_task scheduled__2025-10-03T13:10:00+00:00 [queued]>
[2025-10-03T13:12:07.787+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-03T13:12:07.803+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): select_rows_task> on 2025-10-03 13:10:00+00:00
[2025-10-03T13:12:07.811+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=872) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-03T13:12:07.813+0000] {standard_task_runner.py:63} INFO - Started process 874 to run task
[2025-10-03T13:12:07.813+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_four', 'select_rows_task', 'scheduled__2025-10-03T13:10:00+00:00', '--job-id', '225', '--raw', '--subdir', 'DAGS_FOLDER/dag_four.py', '--cfg-path', '/tmp/tmpmw1yu6q6']
[2025-10-03T13:12:07.816+0000] {standard_task_runner.py:91} INFO - Job 225: Subtask select_rows_task
[2025-10-03T13:12:07.835+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-03T13:12:07.893+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_four.select_rows_task scheduled__2025-10-03T13:10:00+00:00 [running]> on host 7aff8d0a020c
[2025-10-03T13:12:07.992+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_four' AIRFLOW_CTX_TASK_ID='select_rows_task' AIRFLOW_CTX_EXECUTION_DATE='2025-10-03T13:10:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-03T13:10:00+00:00'
[2025-10-03T13:12:07.995+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-03T13:12:07.996+0000] {sql.py:276} INFO - Executing: SELECT * FROM my_table;
[2025-10-03T13:12:08.006+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-10-03T13:12:08.017+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-10-03T13:12:08.023+0000] {sql.py:487} INFO - Running statement: SELECT * FROM my_table;, parameters: None
[2025-10-03T13:12:08.024+0000] {sql.py:496} INFO - Rows affected: 9
[2025-10-03T13:12:08.025+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-03T13:12:08.056+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_four, task_id=select_rows_task, run_id=scheduled__2025-10-03T13:10:00+00:00, execution_date=20251003T131000, start_date=20251003T131207, end_date=20251003T131208
[2025-10-03T13:12:08.110+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-03T13:12:08.126+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-03T13:12:08.132+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
